erpnext/stock/report/itemwise_recommended_reorder_level/itemwise_recommended_reorder_level.py

```diff
@@ -55,14 +55,28 @@ def get_item_info(filters):
 
 
 def get_consumed_items(condition):
+	stock_entry_types_to_exclude = ', '.join([f"'{se}'" for se in [
+		"Material Transfer for Manufacture",
+		"Material Transfer",
+		"Send to Subcontractor"
+	]])
+	condition += f"""
+		and (
+			stock_entry_type is NULL
+			or stock_entry_type not in ({stock_entry_types_to_exclude})
```

Comment:
Use `purpose` instead. Users are allowed to define more stock entry types with the standard set of purposes.

https://docs.erpnext.com/docs/user/manual/en/stock/stock-entry#22-stock-entry-type